lab-ron/server.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+// node modules
+const net = require('net');
+const EE = require('events');
+
+// npm modules
+// app modules
+const Client = require('./model/client.js');
+
+// env vars
+const PORT = process.env.PORT || 3000;
+
+// module constats
+const pool = [];
+const server = net.createServer();
+const ee = new EE();
+
+
+
```

Comment:
@Rubiksron too many spaces here - best practice is to keep a single line break between major components